src/components/common/Extensions/SpecificSidePanel/SpecificSidePanel.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, Fragment } from 'react';
+import { useIntl } from 'react-intl';
+import { SpecificSidePanelProps } from './types';
+import styles from './SpecificSidePanel.module.css';
+import ExtensionStatusBadge from '../ExtensionStatusBadge-new/ExtensionStatusBadge';
+import Permissions from './partials/Permissions';
+import DateInstalled from './partials/DateInstalled';
+import InstaledBy from './partials/InstaledBy';
+import Version from './partials/Version';
+import ContractAddress from './partials/ContractAddress';
+import Developer from './partials/Developer';
+
+const displayName = 'common.Extensions.SpecificSidePanel';
+
```

Comment:
Why are you passing an array of types here? Maybe it will be better to pass current status?

Reply:
you are right, shouldn't be types but statuses. It should be array because there is a case when will be more statuses.